src/main/resources/versions.properties

```diff
@@ -1,14 +1,17 @@
+# This file is copied from https://raw.githubusercontent.com/bonigarcia/webdrivermanager/master/src/main/resources/versions.properties
```

Comment:
Maybe it's better to fetch this file from github url to use latest updates from source repository? And if it isn't available use local file instead?